modules/contract/contracts.js

```diff
@@ -83,15 +83,33 @@ class Contracts {
     async listEntryPoints(contractPath) {
         const fs = require('fs');
         const conseiljs = require(CONSEIL_JS);
+        let conseilServer = { 'url': CONSEIL_SERVER_URL, 'apiKey': CONSEIL_SERVER_APIKEY, 'network': TESTNET_NAME };
+        let contractCode, contractAddress;
+
+        let contractObj = Helper.findKeyObj(config.contracts, contractPath);
+        if (contractObj) {
+            contractAddress = contractObj.pkh;
+        }
+      
+        if (!contractAddress && !fs.existsSync(contractPath)) {
+            Logger.error(`Couldn't find the contract, please make sure contract label or address is correct!`);
+            return;
+        }
 
         try {
-            const contractCode = fs.readFileSync(contractPath, 'utf8');
+            if(contractAddress) {
+                const account = await conseiljs.TezosConseilClient.getAccount(conseilServer, conseilServer.network, contractAddress);
+                contractCode = account.script;
+            } else {
+                contractCode = fs.readFileSync(contractPath, 'utf8');
+            }
+
             const entryPoints = await conseiljs.TezosContractIntrospector.generateEntryPointsFromCode(contractCode);
             const storageFormat = await conseiljs.TezosLanguageUtil.preProcessMichelsonScript(contractCode);
             Logger.info(`\nInitial Storage input must be of type : ${storageFormat[1].slice(8)}`);
             entryPoints.forEach(p => {
                 Logger.info('-------------------------------------------------------------------------------------------------------------------------------------');
-                Logger.info(`\nName => ${p.name}\n\nParameters => (${p.parameters.map(pp => (pp.name || '') + pp.type).join(', ')})\n\nStructure => ${p.structure}\n\nExample => ${p.generateSampleInvocation()}\n`);
+                Logger.info(`\nName => ${p.name}\n\nParameters => (${p.parameters.map(pp => (pp.name || '') + pp.type).join(', ')})\n\nStructure => ${p.structure}\n`);
```

Comment:
is removing 'p.generateSampleInvocation()' an intentional change?

Reply:
yes, because in conseil updated version, this function is breaking down (not giving output) many times for different codes. To prevent from such conflicts I removed that. But for current conseil version we can keep that, let me know we should keep that or remove that?